JapaneseFilter/sections/specific.txt

```diff
@@ -39,6 +39,7 @@ photoshopvip.net,wotopi.jp,mag.osdn.jp,phileweb.com,joqr.co.jp,yutura.net,goethe
 !
 !
 !
+rugby-rp.com##div[class*="-adarea-"]
```

Comment:
Add `rugby-rp.com##.pickup-wrap` too, this is for ad.

Reply:
I will fix

Reply:
done